Cows-HacknSlash/Assets/Scripts/PlayerController/SpellController.cs

```diff
@@ -9,16 +9,22 @@ public class SpellController : MonoBehaviour {
 	[Header("FireBall")]
     [SerializeField]
 	private GameObject FireBall;
+	[Tooltip("0 == null; 1 == FireStaff")]
+	[SerializeField]
+	private int StaffSelected;
     
 	void Update () {
 
         
         if (Input.GetKeyDown(KeyCode.E)){	
-			var projectile = Instantiate(FireBall, SpawnPos.transform.position, Quaternion.identity);
+			
+			if (StaffSelected == 1){
```

Comment:
Can you add a space between the condition and the opening curly brace? (Same for all other if statements)